gui-libs/display-manager-init/files/startDM

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License, v2
+
+# We need to source /etc/profile for stuff like $LANG to work
+# bug #10190.
+. /etc/profile
+
+. /lib/rc/sh/functions.sh
+
+# Bail out early if on a non-OpenRC system:
+if [ ! -d /run/openrc ]; then
+	eerror "$0 should only be used on OpenRC systems"
+fi
+
+# baselayout-1 compat
```

Comment:
I think we do not really need to support baselayout-1 anymore. :)

Reply:
done
removed and retested to make sure nothings gone wrong.